corehq/apps/hqwebapp/static/hqwebapp/yaml/bootstrap5/requirejs.yml

```diff
@@ -12,6 +12,7 @@ fileExclusionRegExp: ^\.|\.css$
 paths:
   sentry_browser: "empty:"
   sentry_captureconsole: "empty:"
+  "tempus-dominus": "empty:"
```

Comment:
This tells requirejs to skip minifying this file (which is already minified). It's necessary because this file uses JS syntax that the requirejs parser can't handle.